### PR TITLE
Xeno landmark changes

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Miscellaneous landmark changes consistency
(Also removed a seemingly unused piece of tunnel code I hope it didn't break everything)

# Explain why it's good for the game

Nest landmarks are legacy from floor nests.
Tunnel structures always spawn, while landmarks only spawn on specific game modes.
It doesn't make sense for a mapped weed node or tunnel to be somewhere where a player could not manually place one.

# Changelog

:cl:
del: Removed nest landmarks
code: Added landmarks for thick versions of xeno structures
code: Added the ability to set hive faction for xeno structure landmarks
code: Added checks for node and tunnel landmarks
maptweak: Replaced all tunnels with tunnel landmarks
maptweak: Movement and deletion of weeds to be able to spread correctly
maptweak: Tunnels slightly moved or turf under and near tunnels changed to allow replacement tunnels to be created.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: maps/map_files/BigRed/BigRed.dmm
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:1]: Mapping seems fine, for some reason in the map editor gibs and glass shards show up as 2x1, probably unrelated.

Most seems to be just find and replace and edits are ratehr minor across the board for tiles/movement of a few landmarks.

**Review by Nanu308 — APPROVED** [review]
Mapping seems fine, for some reason in the map editor gibs and glass shards show up as 2x1, probably unrelated.

Most seems to be just find and replace and edits are ratehr minor across the board for tiles/movement of a few landmarks.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Pull request closed without merging**